src/darwin/mod.rs

```diff
@@ -118,7 +117,7 @@ impl crate::Rasterize for CoreTextRasterizer {
     }
 
     fn load_font(&mut self, desc: &FontDesc, size: Size) -> Result<FontKey, Error> {
-        let scaled_size = Size::new(size.as_f32_pts() * self.device_pixel_ratio);
+        let scaled_size = Size::new(size.as_pt());
```

Comment:
Name is wrong, right? It's just size or ponit size, not scaled size now?

This is also wrong a couple more times in this file, might be worth a quick grep for `scale`.